app/preview/preview.html

```diff
@@ -1,5 +1,6 @@
 <div class="preview-frame"><iframe ng-src="{{ url }}" name="preview-frame"></iframe></div>
 <p ng-if='file'>
-  Previewing {{ file.title }} <a ng-href="{{ url }}" download="filename">(Download)</a>
+  <span translate>Previewing {{ file.title }}</span>
+  <a ng-href="{{ url }}" download="filename">({{ "Download" | translate }})</a>
```

Comment:
Ah, that makes sense. Since you've run into these sorts of issues in AtoM, I'm inclined to follow the same habits.